core/templates/domain/story/StoryObjectFactory.ts

```diff
@@ -245,16 +237,34 @@ export class StoryObjectFactory {
     );
   }
 
-  /**
- * TODO(#14169): Remove the interstitial story so that full story can be
- * created from start.
- */
   // Create an interstitial story that would be displayed in the editor until
   // the actual story is fetched from the backend.
   createInterstitialStory(): Story {
+    let sampleStoryContentsBackendDict = {
+      initial_node_id: '',
+      nodes: [
+        {
+          id: '',
+          title: '',
+          description: '',
+          prerequisite_skill_ids: [''],
+          acquired_skill_ids: [''],
+          destination_node_ids: [''],
+          outline: '',
+          exploration_id: '',
+          outline_is_finalized: false,
+          thumbnail_bg_color: null,
+          thumbnail_filename: null
+        }],
+      next_node_id: ''
+    };
+    let mockStoryContent = StoryContents.createFromBackendDict(
+      sampleStoryContentsBackendDict);
+
     return new Story(
-      null, 'Story title loading', 'Story description loading',
-      'Story notes loading', null, 'en', 1, null, null, null, null, '');
+      'Id loading', 'Story title loading', 'Story description loading',
+      'Story notes loading', mockStoryContent, 'en', 1, 'Topic id loading',
+      null, null, '', '');
```

Comment:
I don't think this is a good solution for the interstitially problem.

Reply:
I have updated the solution as per the suggestion received. PTAL, thanks!